lib/component.ex

```diff
@@ -38,6 +39,13 @@ defmodule LiveSvelte do
     examples: [true, false]
   )
 
+  attr(
+    :socket,
+    :map,
+    default: nil,
+    doc: "Socket to check if the render is live and ssr can be skipped"
```

Comment:
Suggestion:
`The LiveView socket. Should only be provided when rendering inside a LiveView.`